Sources/NostrSDK/Events/BookmarksListEvent.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  BookmarksListEvent.swift
+//
+//
+//  Created by Bryan Montz on 12/22/23.
+//
+
+import Foundation
+
+/// An event that contains an uncategorized, "global" list of things a user wants to save.
+///
+/// See [NIP-51](https://github.com/nostr-protocol/nips/blob/master/51.md#standard-lists).
+public final class BookmarksListEvent: NostrEvent, HashtagInterpreting, PrivateTagInterpreting {
+    public required init(from decoder: Decoder) throws {
+        try super.init(from: decoder)
+    }
+    
+    @available(*, unavailable, message: "This initializer is unavailable for this class.")
+    override init(kind: EventKind, content: String, tags: [Tag] = [], createdAt: Int64 = Int64(Date.now.timeIntervalSince1970), signedBy keypair: Keypair) throws {
+        try super.init(kind: kind, content: content, tags: tags, createdAt: createdAt, signedBy: keypair)
+    }
+    
+    init(content: String, tags: [Tag] = [], createdAt: Int64 = Int64(Date.now.timeIntervalSince1970), signedBy keypair: Keypair) throws {
+        try super.init(kind: .bookmarksList, content: content, tags: tags, createdAt: createdAt, signedBy: keypair)
+    }
+    
+    /// Ids of bookmarked kind-1 notes.
+    public var noteIds: [String] {
+        allValues(forTagName: .event) ?? []
```

Comment:
Similar to my other comment. I think it'll be good to expose relay URL information. It's difficult to discern in my head if it's useful to have both raw string ids like this as well as having a separate one that provide the full gossip information, or just expose only the one for gossip.

Reply:
I think at the library level, it's appropriate to provide both options. I'll rethink this too.